homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Let's have `test_ssdp` finish the test

Reply:
Could you elaborate on this?

Reply:
This is currently the last thing I think should be addressed wrt the quality scale YAML, will do once I understand the request.

Reply:
@joostlek just in case the above went unnoticed, your input re above question is needed here (the final item to address, I hope)